source/reporting/index.html.md.erb

```diff
@@ -261,37 +163,26 @@ If your PSP is Worldpay, contact Worldpay support to find out your payment times
 
 ### PSP fees
 
-If you use GOV.UK's PSP, we'll automatically deduct the PSP's transaction fee from each payment. You can see how much the fee was when you make the API call to get information about a payment.
+If your PSP is Stripe, we'll automatically deduct the PSP's transaction fee from each payment. You can see how much the fee was when you make the API call to get information about a payment.
```

Comment:
what if your PSP isn't Stripe?

Reply:
also has this page spelled out PSP?

Reply:
Good spot on 'PSP' - have added this to the first instance on the page.

Asking devs about what the case is if your PSP isn't Stripe.